joomla/joomla-cms
# [4.0] Fix login alert in footer

Pull Request for Issue #28909.

### Summary of Changes
Move login alert to the top.


### Testing Instructions
Apply PR.
Log in to the backend with wrong credentials.
See that the login alert is at the top and not in the footer.
